endpointGroup.json

```diff
@@ -34,15 +34,35 @@
 					"items": "string",
 					"Title": "Policies",
 					"ShowSummary": true
-				}
+				},
+				"consExtContractsGrps": {
+                                        "type": "array",
+                                        "items": "string",
+                                        "Title": "Consumed external contracts",
+                                        "ShowSummary": true
+                                },
+                                "provExtContractsGrps": {
+                                        "type": "array",
+                                        "items": "string",
+                                        "Title": "Provided external contracts",
+                                        "ShowSummary": true
+                                }
+
 			},
 			"link-sets": {
 				"services": {
 					"ref": "service"
 				},
 				"policies": {
 					"ref": "policy"
-				}
+				},
+                                "consExtContractsGrps": {
```

Comment:
I think we only need one array because we have the type inside the contractGroup. Sorry, I missed this part earlier.